### PR TITLE
Check the following code

### PR DESCRIPTION
**Pull request by developer2401**

### CODE REVIEW

File: code.py [hunk 1]
```diff
@@ -21,59 +21,7 @@
 batch_size = 32
 n_classes = 10
 
-for dirname, _, filenames in os.walk(r'D:\tomato dataset'):   #train dataset address
```
Comment (PrikshitSingh24) [inline code.py:24]: please keep this part.

Comment (PrikshitSingh24) [inline code.py:24]: Please keep this part of the code for the display of the images.

File: code.py [hunk 2]
```diff
@@ -21,16 +21,6 @@
 batch_size = 32
 n_classes = 10
 
-for i, img_path in enumerate(
```
Comment (PrikshitSingh24) [inline code.py:78]: Error in the line, please check .

**Comment by PrikshitSingh24** [comment]
Please recheck the code before requesting for further pulling.

**Review by PrikshitSingh24 — COMMENTED** [review]
you need to change the code

File: code.py (hunk 1 above)
Comment (developer2401) [reply code.py:24]: ok

File: code.py (hunk 2 above)
Comment (developer2401) [reply code.py:78]: I'll recheck.

**Pull request closed without merging**